apps/gradient_autoscheduler/GradientAutoscheduler.cpp

```diff
@@ -535,8 +535,14 @@ void apply_schedule(const MachineParams &params,
         for (int b : rvar_bounds) {
             rdomain_size *= b;
         }
-        int cpu_max_domain_size = 8 * params.parallelism;
-        int gpu_max_domain_size = 4096;
+        // Define the thresholds for the pure domain.
+        // For CPU we want at least params.parallelism number of elements
+        // to launch threads. For GPU we want to launch at least 64 GPU blocks.
+        // We don't use a larger domain size for GPU since we can also use atomic
+        // to increase parallelism and atomics are faster on GPU.
+        // These numbers can be better tuned.
```

Comment:
Does this comment mean that it's worthy of an Issue being opened (and referenced in the comment)?

Reply:
opened and updated comments.